src/main/scala/karazin/scala/users/group/week1/homework/Homework.scala

```diff
@@ -40,27 +41,72 @@ object Homework :
 
   object `Boolean Operators` :
 
-    def not(b: Boolean): Boolean = ???
+    def not(b: Boolean): Boolean = if (b) false else true
 
-    def and(left: Boolean, right: Boolean): Boolean = ???
+    def and(left: Boolean, right: Boolean): Boolean = 
+      if not(left) then

```

Comment:
It's an overkill, it could be easily simplified up to 2 lines.
Hints:
1) get rid of `not` here.
2) if `left` is true what is argument determines the result.

Also `and` should have the same behaviour as Java's `&&`: if `left` is false then `right` should not be calculated.
Hint: use call-by-name strategy.